lms/templates/commerce/checkout_error.html

```diff
@@ -0,0 +1,12 @@
+<%! from django.utils.translation import ugettext as _ %>
+
+<%inherit file="../main.html" />
+
+<%block name="pagetitle">${_("Checkout Error")}</%block>
+
+
+<section class="container">
+    <h1>${_("Checkout Error")}</h1>
+    ${ _(u"An error occurred while processing your payment; you have not been charged. Please try again. If the issue persists, contact {email}.").format(
```

Comment:
Thanks @srpearce. That's totally doable. @tasawernawaz please include a screenshot once this is updated.